docs/flex.md

```diff
@@ -120,3 +120,12 @@ Use `flex-none` to prevent a flex item from growing or shrinking.
   </div>
 </div>
 ```
+
+### Arbitrary values
+If you need to use a one-off `flex` value that doesn’t make sense to include in your theme, use square brackets to generate a property on the fly using any arbitrary value.
```

Comment:
can remove the bit about the theme, like : 
If you need to use a one-off `flex` value, use square brackets to generate a property on the fly using any arbitrary value.

Reply:
Fixed now! Good suggestion. We do have this part of the text in other places as well. We can maybe adjust there as well if it is needed. For example in grid-auto-rows.md.